docs/contributing/governance.md

```diff
@@ -0,0 +1,67 @@
+
+Decisions about Gatsby open-source software are ultimately made by Gatsby, Inc. leadership and the Core Team. We believe that founding organizations built around open source software have a responsibility to build strong businesses to sustain those open source tools and the community that depends on them, which will sometimes require us to reserve high-demand tools and features for Gatsby commercial products. That said, we take our responsibility to the community very seriously and carefully consider community needs and concerns when determining our product roadmaps.
+
+_If you’d like to learn more about how we approach the relationship between our commercial and open source work, check out this blog post by Gatsby co-founder Sam Bhagwat: [Founding Organizations: Creating a Company That Sustains Our Open-Source Community](https://www.gatsbyjs.org/blog/2020-02-11-founding-organizations/#reach-skip-nav)._
+
+### Feature Requests
+
+If you’d like to request or propose a Gatsby feature (or a change to an existing feature), the best way to start is to submit it as an issue via GitHub. The Core Team will then review your submission and decide whether or not to put it through our formal “Request for Comment” process.
+
+### Community PR and Issue Review Process
+
+Issues submitted by the community go through our Triage Process. You can learn more about this process in our [Triaging GitHub Issues doc](https://www.gatsbyjs.org/contributing/triaging-github-issues/).
+
+Our pull request review process is outlined in our [Managing Pull Requests doc](https://www.gatsbyjs.org/contributing/managing-pull-requests/).
+
+If you have any questions about an issue or PR, you can add a comment to the issue/PR on GitHub or reach out to us via one of the options in the "Engage with the Gatsby Team" section below.
+
+## Code of Conduct
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+[Read the full code of conduct](https://www.gatsbyjs.org/contributing/code-of-conduct/)
+
+## Guidance and documentation for contributors
+
+Here are some relevant docs and resources for contributors:
+
+- [Contributing to Gatsby.js](https://www.gatsbyjs.org/contributing/)
+- [How to contribute](https://www.gatsbyjs.org/contributing/how-to-contribute/)
+- [How to file an issue](https://www.gatsbyjs.org/contributing/how-to-file-an-issue/)
+- [Request-for-comment (RFC) process](https://www.gatsbyjs.org/contributing/rfc-process/)
+- [Community Maintainers Meeting](https://www.gatsbyjs.org/contributing/community/#community-maintainers-meeting)
+- [Pair programming sessions](https://www.gatsbyjs.org/contributing/pair-programming/)
```

Comment:
```suggestion
- [Contributing to Gatsby.js](/docs/contributing)
- [How to contribute](/docs/contributing/how-to-contribute/)
- [How to file an issue](/docs/contributing/how-to-file-an-issue/)
- [Request-for-comment (RFC) process](/docs/contributing/rfc-process/)
- [Community Maintainers Meeting](/docs/contributing/community/#community-maintainers-meeting)
- [Pair programming sessions](/docs/contributing/pair-programming/)
```

Reply:
note: the `/contributing*` are not under the `/docs*`

Reply:
> note: the `/contributing*` are not under the `/docs*`

not fixed :(